man/subscription-manager.8

```diff
@@ -396,17 +396,16 @@ The
 command has subcommands for all the various syspurpose preferences and attributes:
 
 .IP
-1. addons
+1. role
```

Comment:
The command is deprecated in this version. Not removed.

Reply:
That matches what we have done in the past for top-level commands. Deprecated ones mentioned below the main list, on one line.

https://github.com/candlepin/subscription-manager/blob/c2551d4df1cf4b38dbcc93848d052021020423f0/man/subscription-manager.8#L134-L141

Reply:
Yeah, but this is list of syspurpose sub-commands and all sub-commands will be still the same in 1.29 branch.